### DISCUSSION TITLE
Is there a way to change "Search Diagnostic" view like telescope?

### DISCUSSION THREAD

**Discussion by snova-paulk** [ibhagwan/fzf-lua#1500] [Q&A]
I'm considering to migrate from telescope to fzf-lua. The only concern is the diagnostic view. Is there a way to set it up?

This is the telescope "Workspace Diagnostic".
<img width="1217" alt="image" src="https://github.com/user-attachments/assets/a58f8438-3f82-4cf9-90b0-486b733c5691">

this is the fzf-lua "Search Diagnostic".
<img width="1181" alt="image" src="https://github.com/user-attachments/assets/3dc3285c-815e-4b5c-827a-c0a21ac855f0">

I don't think the executable python path is needed. I wish to have information shown in the telescope.

**Comment by ibhagwan** [comment]
Changing the format shouldn't be this hard but currently the entries are created similar to grep output (i.e. `filename:col:line:TEXT`), what about using `--with-nth` to hide the filename?
```lua
:FzfLua diagnostics_workspace multiline=false fzf_opts.--delimiter=: fzf_opts.--with-nth=4..
```
